### PR TITLE
added permalink instructions #14

### PR DESCRIPTION
**Pull request by enkidulan**
refs: https://app.zenhub.com/workspace/o/niteoweb/niteoweb.intra/issues/14

### CODE REVIEW

File: managing-docs.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ We use the following services for storing information:
 - **GitHub** - Code repository and ticketing system for all our projects, also hosts this Handbook where we write all company processes that can be publicly published.
 - **[ZenHub](https://www.zenhub.com/)** - Agile project management within GitHub. Adds better predictability and visibility to software development.
 - **[Intra](https://intra.niteoweb.com)** - Project documentation, team members profiles, reports and anything else that cannot be publicly published on the Handbook.
+- **[collective.permalink](https://pypi.python.org/pypi/collective.permalink)** - an extension for **[Intra](https://intra.niteoweb.com)** that creates a persistent link to the Intra's content that should never change even if you rename or move it. Use it every time you need to reference some content outside Intra. To get permalink copy `Permalink (to Clipboard)` link that is on the bottom of the content on every page in Intra.
```
Comment (zupo) [inline managing-docs.md]: `Use it every time you need to reference some Intra content from the outside of Intra, like from the Handbook.`

Comment (zupo) [reply managing-docs.md]: `To get a permalink copy`.

Did you run Grammarly on your changes?

Comment (enkidulan) [reply managing-docs.md]: Yep, I did, and my PREMIUM Grammarly didn't find any issues...

**Comment by enkidulan** [comment]
updated according to review

**Pull request merged**